testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/miniapplist/MiniAppListFragment.kt

```diff
@@ -62,35 +69,97 @@ class MiniAppListFragment : BaseFragment(), MiniAppList {
         viewModel = ViewModelProvider.NewInstanceFactory().create(MiniAppListViewModel::class.java).apply {
             miniAppListData.observe(viewLifecycleOwner, Observer {
                 swipeRefreshLayout.isRefreshing = false
-                displayMiniAppList(it)
+                addMiniAppList(it)
                 MiniAppListStore.instance.saveMiniAppList(it)
             })
             errorData.observe(viewLifecycleOwner, Observer {
                 val list = MiniAppListStore.instance.getMiniAppList()
                 if (list.isEmpty())
-                    Toast.makeText(context, it, Toast.LENGTH_LONG).show()
+                    updateEmptyView(list)
                 else
-                    displayMiniAppList(list)
-                swipeRefreshLayout.isRefreshing = false
+                    addMiniAppList(list)
```

Comment:
This is missing `{}` in the else, the second statement will run differently. What do you think?